src/main/java/net/sf/jabref/logic/importer/fileformat/ModsImporter.java

```diff
@@ -0,0 +1,463 @@
+import net.sf.jabref.logic.importer.fileformat.mods.AbstractDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.DateDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.DetailDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.ExtentDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.GenreDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.HierarchicalGeographicDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.IdentifierDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.IssuanceDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.LanguageDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.LanguageTermDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.LocationDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.ModsCollectionDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.ModsDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.NameDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.NamePartDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.NoteDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.OriginInfoDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.PartDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.PhysicalLocationDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.PlaceDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.PlaceTermDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.RecordInfoDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.RelatedItemDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.StringPlusLanguage;
+import net.sf.jabref.logic.importer.fileformat.mods.StringPlusLanguagePlusAuthority;
+import net.sf.jabref.logic.importer.fileformat.mods.StringPlusLanguagePlusSupplied;
+import net.sf.jabref.logic.importer.fileformat.mods.SubjectDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.TitleInfoDefinition;
+import net.sf.jabref.logic.importer.fileformat.mods.UrlDefinition;
+import net.sf.jabref.logic.util.FileExtensions;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+
+import com.google.common.base.Joiner;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+/**
+ *
+ * Importer for the MODS format.<br>
+ * More details about the format can be found here <a href="http://www.loc.gov/standards/mods/">http://www.loc.gov/standards/mods/</a>. <br>
+ * The newest xml schema can also be found here <a href="www.loc.gov/standards/mods/mods-schemas.html.">www.loc.gov/standards/mods/mods-schemas.html.</a>.
+ *
+ */
+public class ModsImporter extends ImportFormat {
+
+    private static final Log LOGGER = LogFactory.getLog(ModsImporter.class);
+    private static final String KEYWORD_SEPARATOR = "; ";
```

Comment:
If I use the keyword separator from the preferences and I have keyword1 and keyword2, I've got ``keyword1groupKeywordSeparatorkeyword2". Is this how it is wanted?